translations/messages.fr.xlf

```diff
@@ -0,0 +1,3065 @@
+      </trans-unit>
+      <trans-unit id="1204343429877937229" datatype="html">
+        <source>Nice, you have disconnected 4 pieces of your opponent!
+        He has lost 4 points.
+        Disconnected stacks will not be visible at the next turn.</source>
+        <target>Bravo, vous avez déconnecté 4 pièces de l’adversaire !
+        Il a donc perdu 4 points.
+        Les piles déconnectées cesseront d&apos;être visibles au tour suivant.</target>
+      </trans-unit>
+      <trans-unit id="9072861571041891023" datatype="html">
+        <source>Bad choice! By moving on the source you would have disconnected the opponent&apos;s pieces and they would have lost 4 points.
+        Here, the opponent wins 2 to 0</source>
+        <target>Mauvais choix! En le déplaçant sur la source vous déconnectiez l&apos;adversaire et lui faisiez perdre ces 4 points.
+        Ici, il gagne 2 à 0.</target>
+      </trans-unit>
+      <trans-unit id="3483594998155999756" datatype="html">
+        <source>When no more move is possible, the game ends and the player with the most points wins.&lt;br/&gt;&lt;br/&gt;
+        Make your last move.</source>
+        <target>Quand plus aucun mouvement n’est possible, la partie est finie et le joueur avec le plus de points gagne.&lt;br/&gt;&lt;br/&gt;
+        Faites votre dernier mouvement !</target>
+      </trans-unit>
+      <trans-unit id="2963709509031109432" datatype="html">
+        <source>Congratulations, you won 6 - 0!</source>
+        <target>Bravo, vous avez même gagné 6 - 0 !</target>
+      </trans-unit>
+      <trans-unit id="8876232297721386956" datatype="html">
+        <source>Bad idea, by moving on the source you would have won a point.</source>
+        <target>Mauvaise idée, en la déplaçant sur la source, vous auriez gagné votre pièce et gagné un point.</target>
+      </trans-unit>
+      <trans-unit id="6986218395331151516" datatype="html">
+        <source>You must pick one of your pieces.</source>
+        <target>Veuillez utiliser une de vos pièces.</target>
+      </trans-unit>
+      <trans-unit id="6059738106874378452" datatype="html">
+        <source>You do not have pieces of this type anymore.</source>
+        <target>Vous n&apos;avez plus de pièces de ce type.</target>
+      </trans-unit>
+      <trans-unit id="2129733726620651846" datatype="html">
+        <source>You must put your piece on an empty square or on a smaller piece.</source>
+        <target>Vous devez placer votre pièce sur une case vide ou sur une pièce plus petite.</target>
+      </trans-unit>
+      <trans-unit id="5649666705061470825" datatype="html">
+        <source>You must pick your piece among the remaining ones.</source>
+        <target>Veuillez choisir une de vos pièces parmi les pièces restantes.</target>
+      </trans-unit>
+      <trans-unit id="7757464288692634777" datatype="html">
+        <source>You must pick one of your piece or one square where you have the largest piece.</source>
```

Comment:
"You must pick one of your remaining piece or one piece on the board that is the biggest of its space."
Non ?

Reply:
Ou bien, "or one **space** where you"-blbl
En effet très suffisant !

Reply:
Ah non, square et space sont équivalemment bien en anglais c'est ça ? 
Donc space est utilisé pour les non-square square :p ?

Reply:
square = case carrée, space = tout le reste oui ! Du coup ici, on part sur ta première proposition en remplaçant space par square ?

Reply:
"You must pick one of your remaining piece or one piece on the board that is the biggest of its square."
Oui !

Reply:
Done!